### ISSUE TITLE
Guard against file truncation in CalcJobOutputFollower

### ISSUE THREAD

**Issue by danielhollas** [ispg-group/aiidalab-ispg#223]
We need to put  a bounds check on this line:

https://github.com/ispg-group/aiidalab-ispg/blob/54081f89ff186793cae8a754fb1746ccf29c9979/aiidalab_ispg/app/qeapp/widgets.py#L320

Same bug is present in QeApp CC @unkcpz

**Comment by unkcpz** [comment]
So you are worried about the lines being too many? What is the change you proposed?

**Comment by danielhollas** [comment]
Yes, if the file got truncated somehow than the code I linked to will raise an out-of-bounds error. We need to add a check to prevent that.

I'll prepare a PR for my app and ask you for review.
